deploy/examples/csi/rbd/storageclass-radosnamespace-test.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: ceph.rook.io/v1
```

Comment:
Let's rename this file without the "test" suffix. I think no need to commit a test version of this file. It won't be common to use during development. Normally the "test" file are only used during development.

Reply:
done